package/kedro_viz/server.py

```diff
@@ -210,7 +214,9 @@ def _call_viz(host=None, port=None, browser=None, load_file=None, save_file=None
                 click.echo("Invalid file, top level key '{}' not found.".format(key))
                 sys.exit(1)
     else:
-        data = get_data_from_kedro()
+        pipeline = get_project_context("create_pipeline")()
+        catalog = None  # currently unused variable for future proofing
```

Comment:
`get_project_context("create_catalog")()`

Reply:
this is what it will be in future, best to do it right now so that anyone who copies this sees the future proof way